File: back/wildcircus/src/main/java/com/checkpoint/wcs/wildcircus/entities/Performance.java
package com.checkpoint.wcs.wildcircus.entities;

import jdk.internal.perf.Perf;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

@Entity
public class Performance {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    private String title;
    private String content;

    public Performance() { }

    public Performance(String title, String content) {
        this.title = title;
        this.content = content;
    }

    public Long getId() {
        return id;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getContent() {
        return content;
    }

    public void setContent(String content) {
        this.content = content;
    }
}

File: front/src/app/models/performance.ts
export class Performance {
    private title: string;
    private content: string;

    constructor(title: string, content: string) {
        this.title = title;
        this.content = content;
    }

    public getTitle() {
        return this.title;
    }

    public getContent() {
        return this.content;
    }
}
